server/Cargo.lock

```diff
@@ -0,0 +1,56 @@
+[root]
```

Comment:
`Cargo.lock` records the exact versions of all the dependencies used for the previous build, and if it already exists it will not be overwritten without explicitly running `cargo update`.

The general philosophy behind Cargo is that the end-result binaries control how their dependencies are built and used, so the standard way to use `Cargo.lock` is to check it in for applications and to `.gitignore` it for libraries. This way application builds are reproducible across machines.